src/main/java/de/retest/recheck/RecheckOptions.java

```diff
@@ -175,8 +174,8 @@ public RecheckOptionsBuilder fileNamerStrategy( final FileNamerStrategy fileName
 		/**
 		 * @param namingStrategy
 		 *            The {@link NamingStrategy} that determines how to name tests and suites. Default is
-		 *            {@link ClassAndMethodBasedNamingStrategy}. Other options include
-		 *            {@link ClassAndMethodBasedShortNamingStrategy} and {@link ExplicitMutableNamingStrategy}.
+		 *            {@link JunitbasedNamingStrategy}. Other options include {@link JunitbasedShortNamingStrategy} and
```

Comment:
The class names have changed in the meantime, see diff.

Reply:
Here you still use the old class name, which fails the Javadoc generation: https://travis-ci.com/retest/recheck/builds/128311562#L1287